readthedocs/projects/forms.py

```diff
@@ -166,6 +166,7 @@ class Meta:
         fields = (
             'description',
             'documentation_type',
+            'default_branch',
```

Comment:
Is this to allow users to edit this in the Project edit form, or is it required for the `import.js` to pass it properly?

Reply:
Ah, this is on import. I'm 👍  on showing it only when users select "Edit advanced options" 👍

Reply:
For both, yeah p: